inference-engine/tests/functional/plugin/gna/pass_tests/convert_padded2valid_conv.cpp

```diff
@@ -277,14 +277,14 @@ const std::vector<std::vector<size_t >> maxpool1DPools = { {1, 2} };
 const std::vector<std::vector<size_t >> maxpool1DStrides = { {1, 1} };
 
 const std::vector<std::vector<size_t>> input2DNHWC = { {1, 16, 16, 32} };
-const std::vector<std::vector<size_t >> kernels2D = { {2, 2}, {4, 1}, {1, 3}};
+const std::vector<std::vector<size_t >> kernels2D = { {2, 2}, {4, 1}, {1, 3} };
 const std::vector<std::vector<size_t >> strides2D = { {1, 1}, {1, 2}, {2, 1}, {2, 2} };
 const std::vector<std::vector<ptrdiff_t>> padBegins2D = { {1, 2} };
 const std::vector<std::vector<ptrdiff_t>> padEnds2D = { {3, 1} };
 const std::vector<std::vector<size_t >> dilations2D = { {1, 1} };
-const std::vector<size_t> numOutChannels2D = { 32 };
-const std::vector<std::vector<size_t >> biases2D = { {1, 32, 1, 1} };
-const std::vector<std::vector<size_t >> transpBiases2D = { {1, 1, 1, 32} };
+const std::vector<size_t> numOutChannels2D = { 8 };
+const std::vector<std::vector<size_t >> biases2D = { {1, 8, 1, 1} };
+const std::vector<std::vector<size_t >> transpBiases2D = { {1, 1, 1, 8} };
```

Comment:
The change to 8 was done, because on an earlier stage there were some issues with 8 channels. These test parameters are dependent on each other, so a combination of 8 and 32 cannot be used. So I'll stick to 8 as a better choice to find potential issues.